Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <PackageId>EdFi.Admin.DataAccess</PackageId>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Please upgrade package references; you can try the https://dotnet.microsoft.com/en-us/platform/upgrade-assistant or manually do it. No need to upgrade EdFi.Suite3.Common yet.